client/src/views/devices/GreetDeviceModal.vue

```diff
@@ -36,26 +36,71 @@
         <!-- waiting step -->
         <div
           v-show="pageStep === GreetDeviceStep.WaitForGuest"
-          class="step"
+          class="first-step"
         >
-          <ms-informative-text>
-            {{ $t('DevicesPage.greet.subtitles.waitForGuest1') }}
-          </ms-informative-text>
-          <ms-informative-text>
-            {{ $t('DevicesPage.greet.subtitles.waitForGuest2') }}
-          </ms-informative-text>
-          <ion-buttons slot="start">
-            <ion-button
-              fill="solid"
-              size="default"
-              id="copy-link"
-              @click="copyLink"
-            >
-              <span>
-                {{ $t('DevicesPage.greet.copyLink') }}
-              </span>
-            </ion-button>
-          </ion-buttons>
+          <!-- title -->
+          <p class="first-step__title">
+            <ion-text class="first-step__title__blue">
+              {{ $t('DevicesPage.greet.subtitles.scanOrShare1') }}
+            </ion-text>
+            <ion-text class="first-step__title__grey">
+              {{ $t('DevicesPage.greet.subtitles.scanOrShare2') }}
+            </ion-text>
+            <ion-text class="first-step__title__blue">
+              {{ $t('DevicesPage.greet.subtitles.scanOrShare3') }}
+            </ion-text>
+          </p>
+          <!-- qr code, link and button -->
+          <div class="elements">
+            <!-- left element: qr code -->
+            <vue-qrcode
+              class="first-step__qrcode"
+              :type="'image/png'"
```

Comment:
Since the prop is a const string, you can just use `type="image/png"`. Makes it slightly more readable by not having the double quotes.